simulcast.html

```diff
@@ -0,0 +1,410 @@
+    this specification MUST implement them in a manner consistent with the
+    ECMAScript Bindings defined in the Web IDL specification [[!WEBIDL-1]], as
+    this specification uses that specification and terminology.</p>
+  </section>
+  <section>
+    <h2>Terminology</h2>
+     <p>The <code><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#eventhandler">EventHandler</a></code>
+      interface, representing a callback used for event handlers, and the <a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#errorevent"><code><dfn>ErrorEvent</dfn></code></a>
+      interface are defined in [[!HTML51]].</p>
+      <p>The concepts <dfn><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#queue-a-task">queue a task</a></dfn>,
+      <dfn><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#fire-a-simple-event">fires a simple
+      event</a></dfn> and <dfn><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#networking-task-source">networking
+      task source</a></dfn> are defined in [[!HTML51]].</p>
+      <p>The terms <dfn>event</dfn>, <dfn><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#event-handlers">event
+      handlers</a></dfn> and <dfn><a href=
+      "http://dev.w3.org/html5/spec/webappapis.html#event-handler-event-type">event
+      handler event types</a></dfn> are defined in [[!HTML51]].</p>
+     <p>When referring to exceptions, the terms <dfn><a
+      href="https://www.w3.org/TR/WebIDL-1/#dfn-throw">throw</a></dfn> and
+      <dfn data-dfn-for="exception"><a href=
+      "https://www.w3.org/TR/WebIDL-1/#dfn-create-exception">create</a></dfn> are
+      defined in [[!WEBIDL-1]].</p>
+      <p>The terms <dfn data-lt="fulfill|fulfillment">fulfilled</dfn>, <dfn
+      data-lt="reject|rejection|rejecting">rejected</dfn>,
+      <dfn data-lt="resolve|resolves">resolved</dfn>, <dfn>pending</dfn> and
+      <dfn>settled</dfn> used in the context of Promises are defined in
+      [[!ECMASCRIPT-6.0]].</p>
+      <p>The terms <dfn>MediaStream</dfn>, <dfn>MediaStreamTrack</dfn>, and
+      <dfn>MediaStreamConstraints</dfn> are defined in [[!GETUSERMEDIA]].</p>
+      <p>This specification references objects, internal slots and dictionaries defined in
+      [[!WEBRTC]], including the <dfn><code>RTCRtpEncodingParameters</code></dfn> dictionary
+      (defined in Section 5.2.6), the <dfn><code>RTCRtpTransceiver</code></dfn> object,
+      its <dfn>[[\Sender]]</dfn> and <dfn>[[\Receiver]]</dfn> internal slots (defined
+      in Section 5.4), the <dfn><code>RTCRtpSender</code></dfn> object, its
+      <dfn>[[\SendEncodings]]</dfn> and <dfn>[[\LastReturnedParameters]]</dfn> internal slots (defined in Section 5.2),
+      and the <dfn><code>RTCRtpReceiver</code></dfn> object (defined in Section 5.3).</p>
+  </section>
+  <section id="rtcrtpcodingparameters">
+    <h3><dfn>RTCRtpCodingParameters</dfn> Dictionary</h3>
```

Comment:
Does RTCRtpParameters, which contains RTCRtpCodecParameters, now need to be updated to reference this document? Or does the codecs member become an extension defined here for RTCRtpParameters?